backend/PyMatcha/__init__.py

```diff
@@ -71,8 +71,8 @@
 application.config["JWT_SECRET_KEY"] = os.environ.get("FLASK_SECRET_KEY")
 
 logging.debug("Configuring Celery Redis URLs")
-CELERY_BROKER_URL = "redis://localhost:6379/0" if not os.getenv("IS_DOCKER_COMPOSE") else "redis://redis:6379/0"
-CELERY_RESULT_BACKEND = "redis://localhost:6379/0" if not os.getenv("IS_DOCKER_COMPOSE") else "redis://redis:6379/0"
+CELERY_BROKER_URL = os.getenv("CELERY_BROKER_URL", "redis://localhost:6379/0")
```

Comment:
You are using `os.getenv("REDIS_HOST", "localhost")` on line 154, could it also be used here and also for the port number?

Reply:
But then I don't know if it's bad or not to have os.getenv in os.getenv. Maybe it's too much of an inception haha

Reply:
haha yeah it's a bad thing ;) The idea is that `CELERY_BROKER_URL` is also a redis URI but a separate one so it's set appart in the .env file